Xcode/PDFImage/PDFImageView.m

```diff
@@ -61,6 +61,17 @@ - (instancetype)initWithFrame:(CGRect)frame
 	return self;
 }
 
+- (instancetype)initWithImage:(PDFImage *)image {
```

Comment:
I'm being picky but please move the brace to a new line to be consistent with the other methods